### DISCUSSION TITLE
Feature request: File source pause, fast forward, reverse separate waterfall

### DISCUSSION THREAD

**Discussion by janvgils** [AlexandreRouma/SDRPlusPlus#780] [Ideas]
Good day,

Thanks for taking the time to read this new discussion.
One of the really great features when using the SDR++ is the ability to playback an IQ recording.

When analyzing the recording it would be great if it was possible to step through this recording if possible in a waterfall overview.

![image](https://user-images.githubusercontent.com/21240133/172056717-843986ae-9dfa-4d15-a259-3ad320ff62be.png)

With regards,

Jan PE0SAT

**Comment by the-gobleo-tinkerer** [comment]
The overview feature was requested in #129. Sadly it won't be added very soon, unless someone else writes it.
